tests/rfc003_erc20_htlc.rs

```diff
@@ -45,10 +47,13 @@ fn given_erc20_token_should_deploy_erc20_htlc_and_fund_htlc() {
         gas_limit: U256::from(100_000),
         to: Some(token_contract),
         value: U256::from(0),
-        data: Some(Erc20Htlc::transfer_erc20_tx_payload(
-            token_amount,
-            htlc_address,
-        )),
+        data: Some(
+            Erc20Htlc::transfer_erc20_tx_payload(
+                TokenQuantity(token_amount.into()),
```

Comment:
Can this be written more cleanly with a `from` method?

Reply:
Nope as none of the `struct` are defined in the test module.
If we were to define the `From` in the crate then we would have to pull in the web3 dep which is what we are trying to avoid :)